### PR TITLE
Added device type API python sample

### PR DESCRIPTION
**Pull request by GDLMadushanka**
python sample code for creating device types from API

### CODE REVIEW

File: DeviceTypeAPI_PythonSample/importer.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+def import_or_install(package):
+    try:
+        __import__(package)
+        print('already there')
+    except ImportError:
+        pip.main(['install', package])
+
+def installMissingPackages():
+    os.system('pip install -Iv paho-mqtt==1.2.3')
```
Comment (Megala21) [inline DeviceTypeAPI_PythonSample/importer.py]: Add a extra line in the file-end

File: DeviceTypeAPI_PythonSample/READ_ME [hunk 2]
```diff
@@ -0,0 +1,17 @@
+READ ME
```
Comment (Megala21) [inline DeviceTypeAPI_PythonSample/READ_ME]: Normally 'README' is a one word

File: DeviceTypeAPI_PythonSample/READ_ME [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+Sample MQTT clients for Creating device type via API (WSO2 IOTS 3.1)
```
Comment (Megala21) [inline DeviceTypeAPI_PythonSample/READ_ME]: WSO2 IOT 3.1.0 / 3.1.1?

File: DeviceTypeAPI_PythonSample/READ_ME [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+Connects with the initial token which has a validity period of 1hr. 
+Will not work if disconnetcs and connects again after token expired.
+If client maintains the connection continously, It will continue to be connected even after 1hr
+
+SampleClient2 :- 
+Take new access token each time it begins the execution. 
+So token expiration not be an issue
+
+Replace your config file with config.json file inside the Configuration folder
+
```
Comment (Megala21) [inline DeviceTypeAPI_PythonSample/READ_ME]: May be we can add the steps to start with?

**Review by Megala21 — CHANGES_REQUESTED** [review]

File: DeviceTypeAPI_PythonSample/READ_ME (hunk 4 above)
Comment (GDLMadushanka) [reply DeviceTypeAPI_PythonSample/READ_ME]: Steps are there in the documentation. (Link provided) May I remove last sentence if it is confusing ?

File: DeviceTypeAPI_PythonSample/Configuration/.gitignore [hunk 5]
```diff
@@ -0,0 +1,3 @@
+*
+!.gitignore
```
Comment (dakshika) [inline DeviceTypeAPI_PythonSample/Configuration/.gitignore:2]: Wondering why this contains only this?

File: DeviceTypeAPI_PythonSample/README [hunk 6]
```diff
@@ -0,0 +1,15 @@
+README
+
+Sample MQTT clients for Creating device type via API (WSO2 IOTS 3.1.0 / 3.1.1)
+
+Docs :- https://docs.wso2.com/display/IoTS310/Creating+a+Fire+Alarm+Device+Type+via+the+Console
```
Comment (dakshika) [inline DeviceTypeAPI_PythonSample/README:5]: Incorrect doc URL?

File: DeviceTypeAPI_PythonSample/Configuration/.gitignore (hunk 5 above)
Comment (GDLMadushanka) [reply DeviceTypeAPI_PythonSample/Configuration/.gitignore:2]: User should add the downloaded config file from IOT server, to this folder and run the python code.

File: DeviceTypeAPI_PythonSample/README (hunk 6 above)
Comment (GDLMadushanka) [reply DeviceTypeAPI_PythonSample/README:5]: This is the correct URL. In docs we provide the code as downloadable .zip file and github link which points to this repo

**Pull request merged**